8.2/alpine3.16/fpm-nginx/Dockerfile

```diff
@@ -0,0 +1,130 @@
+#
+# NOTE: THIS DOCKERFILE IS GENERATED VIA "apply-templates.sh"
+#
+# PLEASE DO NOT EDIT IT DIRECTLY.
+#
+
+FROM php:8.2.6-fpm-alpine3.16
```

Comment:
This image doesn't build as Alpine 3.16 isn't available for 8.2 tag it seems

https://hub.docker.com/_/php/tags?page=1&name=8.2.6-fpm-alpine

Maybe we should go to 3.18 for all for security. If you look at the tags [filtered down to 8.1.19](https://hub.docker.com/_/php/tags?page=1&name=8.1.19-fpm-alpine) 3.16 is available but has security vulnerabilities, as does 3.17. 3.18 however is _clean_ ✨ 

----

Looks like 8.1 and 8.2 have Alpine 3.18 versions **but _8.0_ only has 3.16** so we keep 8.0 at 3.16 and all the more reason to get users to upgrade to the newer PHP version.

so

- 8.0 == Alpine 3.16
- 8.1 and 8.2 == Alpine 3.18

Reply:
Done.